README.md

```diff
@@ -4,4 +4,5 @@ fcrepo-audit
 [![Build Status](https://travis-ci.org/fcrepo4-exts/fcrepo-audit.png?branch=master)](https://travis-ci.org/fcrepo4-exts/fcrepo-audit)
 
 
-Provides audit functionality for the Fedora Commons repository framework. The [fcrepo-webapp-plus](https://github.com/fcrepo4-exts/fcrepo-webapp-plus) project provides a convenient option to build fcrepo-webapp packaged with fcrepo-audit module.
\ No newline at end of file
+Provides audit functionality for the Fedora Commons repository framework. The [fcrepo-webapp-plus](https://github.com/fcrepo4-exts/fcrepo-webapp-plus) project provides a convenient option to build fcrepo-webapp packaged with fcrepo-audit module.
+This functionality has been deprecated as of v.4.7.5.  It will no longer continue to be supported by 5.x.
```

Comment:
I think my preference would be to have this line  as the very first thing on this page under build status.  Maybe in bold as well, so it stands out.

Reply:
@bseeger : agreed